exercises/concept/vehicle-purchase/.meta/exemplar.R

```diff
@@ -0,0 +1,17 @@
+needs_license <- function(vehicle) {
+  vehicle %in% c("car", "truck")
```

Comment:
Seeing that both here and in the implementation for `choose_vehicle()` you've made use of `c()` makes me wonder whether maybe it would be better if the vectors concept were introduced before conditionals?

Of course both `needs_license()` and `choose_vehicle()` can be implemented using scalar values plus if/else, so that could be done here but it wouldn't be idiomatic R code.

Reply:
I've pushed a new version of `exemplar.R` that removes all use of vectors. Is it horribly unidiomatic? `needs_license` gets a bit longer and less elegant, but `choose_vehicle` is slightly shorter if given multiple arguments without wrapping them in `c()`.

I also extended `introduction.md` to mention `max()` and `min()` with multiple arguments. If you're happy with this, we'd need to propagate it to `concepts/conditionals`.

Reply:
Alternatively, to keep `%in% c()` we could add `vectors` as a prerequisite for this exercise. It involves some jumping around the concept tree, but I think that's pretty normal for exercises that are not the first in their concept.

Reply:
I don't think we need to make vectors a prerequisite. `needs_license` seems ok because there are only 2 options - it would be worse if there were multiple. So happy to keep as is for now.